doc/rst/usingchapel/chplenv.rst

```diff
@@ -656,6 +656,13 @@ CHPL_UNWIND
 
    If unset, ``CHPL_UNWIND`` defaults to ``none``
 
+.. tip::
+
+   The ``CHPL_RT_UNWIND`` environment variable can be set to ``0``, ``false``, or
+   ``no`` to squash the output that would normally be generated on ``halt()`` when
+   ``CHPL_UNWIND`` is set to something other than ``none``.
```

Comment:
Thanks for taking this on!  As motivation/rationale, I might add something like:  "This can be a useful way to suppress unwind output for a given program without having to re-build the runtime for CHPL_UNWIND=none and/or recompiling the program itself."